ThirdParty/README.md

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
This should read:
    # Third Party
    Sample apps demonstrating common third party frameworks scenarios.